test/src/test/java/org/corfudb/integration/LogReplicationIT.java

```diff
@@ -1380,6 +1377,11 @@ private void verifyPersistedLogEntryMetadata() {
         assertThat(expectedAckTimestamp == lastLogProcessed).isTrue();
     }
 
+    private void verifyRemainingReplicationPercent(long remainingPercent) {
```

Comment:
Where is this being used?

Reply:
I did not remove so that it can be used later when we add tests